app/src/main/java/com/example/soundcloud/selected_genre_detail/SelectedGenreActivity.java

```diff
@@ -16,6 +16,8 @@
     private TextView mTextViewGenreTitle;
```

Comment:
Theo anh em có thể đặt tên ngắn hơn, k phải nhất thiết dài quá :)
```suggestion
    private TextView mTextGenreTitle;
```

Reply:
Vâng. em sửa nó ngay giờ